lib/active_model/serializer.rb

```diff
@@ -27,12 +27,12 @@ class Serializer
       )
     /x
 
-    class << self
-      attr_accessor :_attributes
-      attr_accessor :_attributes_keys
-    end
-
     with_options instance_writer: false, instance_reader: false do |serializer|
+      class_attribute :_type, instance_reader: true
+      class_attribute :_attributes
+      self._attributes ||= []
+      class_attribute :_attributes_keys
```

Comment:
@bf4 I think he prefers having all the `class_attribute` grouped together, and then the `stuff ||= stuff`. I personally do not have an opinion on the matter.